reader-shp/src/main/java/com/graphhopper/reader/shp/OSMShapeFileReader.java

```diff
@@ -53,13 +50,12 @@
  * @author Philip Welch
  */
 public class OSMShapeFileReader extends ShapeFileReader {
-    private static final int COORD_STATE_UNKNOWN = -1;
+    private static final int COORD_STATE_UNKNOWN = 0;
```

Comment:
empty value in the hashmap is now 0 ... maybe replace this with a contains check? But as we do not use 0 this should be fine too